keyboards/duck/octagon/v1/info.json

```diff
@@ -6,7 +6,92 @@
     "height": 6, 
     "layouts": {
         "LAYOUT": {
```

Comment:
Why not just use LAYOUT_75_ansi

Reply:
Yeah, this was a round-about way of going, but I didn't know that going in. I had to go digging through Duck's blog archives to find good images of the V1 PCB.

If I had known the results going in, I might have assigned the macro as LAYOUT_75_ansi and aliased LAYOUT to it. Current state is functionally identical, but the assignments are the other way around.

Reply:
I would prefer just LAYOUT_75_ansi